lib/src/signed_video_h26x_auth.c

```diff
@@ -742,6 +704,8 @@ validate_authenticity(signed_video_t *self)
   } else {
     if (self->gop_info->signature_hash_type == DOCUMENT_HASH) {
       verify_success = verify_hashes_with_sei(self, &num_expected_nalus, &num_received_nalus);
+      // Set |latest_validated_gop| to recived gop counter for the next validation.
+      self->gop_info->latest_validated_gop = self->gop_info->global_gop_counter;
```

Comment:
There might be a conflict between operations. Here we SET `latest_validated_gop` whereas on line 653 above we INCREMENT it. Can you think of any case where this can cause unsynchronized validation for example?

Reply:
We’re only incrementing |latest_validated_gop |while verifying without SEI, which means we only increase it when we’re out of sync. I haven’t thought of any case where this could lead to desynchronization after setting |self->gop_info->latest_validated_gop = self->gop_info->global_gop_counter|.

Reply:
Can this situation cause any problems, where the second `S` is lost?
`IPPISPPPIPPPPIPSPPISP`
Or if we lose 2 SEIs?

Reply:
the validation will be 
 IPPISPPPIPPPPIPSPPISP
 IPPIS                                                          -> ...P.
       ISPPPIPPPPIPS                                   ->N.NNN.....PP.(NOT_OK)
                            IPSPPIS                           ->.....P.
Wich I think its true.